### PR TITLE
update sample readme.md

### PR DESCRIPTION
**Pull request by wswind**
**What issue does this PR address?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [x] The commit follows our [guidelines](https://github.com/IdentityServer/IdentityServer4/blob/master/.github/CONTRIBUTING.md)
- [x] Unit Tests for the changes have been added (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: samples/Quickstarts/readme.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This quickstart adds support for Google authentication.
 
 ## Quickstart #5: OpenID Connect Hybrid Flow Authentication and API Access Tokens
 
-This quickstart adds support for Google authentication.
+This quickstart adds support for hybird flow authentication.
```
Comment (brockallen) [inline samples/Quickstarts/readme.md]: (sp) hybrid*

**Comment by wswind** [comment]
image
the readme document is still wrong

**Comment by leastprivilege** [comment]
..and you never fixed your incorrect spelling. Do an update and I can merge.

**Comment by wswind** [comment]
sorry for the spell error.I fixed the spell in a new pr #3525

**Pull request closed without merging**